docs/tempo/website/getting-started/_index.md

```diff
@@ -5,9 +5,79 @@ weight: 100
 
 # Getting started with Tempo
 
+Distributed tracing is used to visualise the lifecycle of a request as it passes through
+an application. There are a few components that need to be set up in order to get a
+working distributed tracing visualisation.
+
+## 1. Client instrumentation
+
+#### OpenTelemetry Instrumentation SDKs
+
+The first building block to a functioning distributed tracing visualization pipeline
+is client instrumentation, which is the process of adding instrumentation points in the
+application that collect telemetry information. 
+
+There are SDKs available in most commonly used programming languages that can be used
+in this step. Commonly used SDKs are listed below -
+
+* [OpenTemeletry CPP](https://github.com/open-telemetry/opentelemetry-cpp)
+* [OpenTemeletry Java](https://github.com/open-telemetry/opentelemetry-java)
+* [OpenTemeletry JS](https://github.com/open-telemetry/opentelemetry-js)
+* [OpenTemeletry .NET](https://github.com/open-telemetry/opentelemetry-dotnet)
+* [OpenTemeletry Lambda](https://github.com/open-telemetry/opentelemetry-lambda)
+* [OpenTemeletry Go](https://github.com/open-telemetry/opentelemetry-go)
+* [OpenTemeletry Python](https://github.com/open-telemetry/opentelemetry-python)
+* [OpenTemeletry Ruby](https://github.com/open-telemetry/opentelemetry-ruby)
+* [OpenTemeletry Swift](https://github.com/open-telemetry/opentelemetry-swift)
+* [OpenTemeletry Ruby](https://github.com/open-telemetry/opentelemetry-ruby)
+* [OpenTemeletry PHP](https://github.com/open-telemetry/opentelemetry-php)
+* [OpenTemeletry Rust](https://github.com/open-telemetry/opentelemetry-rust)
+
+#### OpenTelemetry Auto Instrumentation
+
+Some languages have support for auto-instrumentation. These libraries capture telemetry
+information from client application with minimal manual instrumentation of the codebase.
+
+* [OpenTemeletry Java Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation)
+* [OpenTemeletry .NET Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
+* [OpenTemeletry Python Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib)
+
+> Note: Check out our [instrumentation examples]() to learn how to instrument your
+> favourite language for distributed tracing.
+
+## 2. Grafana Agent
```

Comment:
I would propose explaining why it's important to have this piece in your pipeline (collector/agent) but trying to keep the section tool agnostic. At least the title and the first paragraph. 

Then, recommending the most used ones: Grafana Agent and OpenTelemetry Collector.

Reply:
Yeah to be honest I was having trouble giving a suitable title to this section, `pipeline`? `agent`? , open to suggestions!